src/sfizz/Voice.cpp

```diff
@@ -34,6 +35,10 @@ sfz::Voice::Voice(int voiceNumber, sfz::Resources& resources)
         filter.setGain(vaGain(config::filteredEnvelopeCutoff, sampleRate));
 }
 
+sfz::Voice::~Voice()
```

Comment:
Any reason to have this?

Reply:
This was a recommendation of google c++ guidelines.
This would ensure destructions don't get inlined in the caller, for economy of code size.